content/docs/rfcs/1/README.md

```diff
@@ -84,56 +106,106 @@ A specification has six possible states that reflect its maturity and contractua
 
 ## Branching and Merging
 
-Any member of the domain MAY branch a specification at any point.  This is done by copying the existing text, and creating a new specification with the same name and content, but a new number.  The ability to branch a specification is necessary in these circumstances:
+Any member of the domain MAY branch a specification at any point.
+This is done by copying the existing text, and creating a new specification with the same name and content, but a new number.
+The ability to branch a specification is necessary in these circumstances:
 
 * To change the responsible editor for a specification, with or without the cooperation of the current responsible editor.
-* To rejuvenate a specification that is stable but needs functional changes.  This is the proper way to make a new version of a specification that is in stable or deprecated status.
+* To rejuvenate a specification that is stable but needs functional changes.
+  This is the proper way to make a new version of a specification that is in stable or deprecated status.
 * To resolve disputes between different technical opinions.
 
 The responsible editor of a branched specification is the person who makes the branch.
 
-Branches, including added contributions, are derived works and thus licensed under the same terms as the original specification.  This means that contributors are guaranteed the right to merge changes made in branches back into their original specifications.
+Branches, including added contributions, are derived works and thus licensed under the same terms as the original specification.
+This means that contributors are guaranteed the right to merge changes made in branches back into their original specifications.
 
-Technically speaking, a branch is a *different* specification, even if it carries the same name.  Branches have no special status except that accorded by the community.
+Technically speaking, a branch is a *different* specification, even if it carries the same name.
+Branches have no special status except that accorded by the community.
 
 ## Conflict resolution
 
-COSS resolves natural conflicts between teams and vendors by allowing anyone to define a new specification.  There is no editorial control process except that practised by the editor of a new specification.  The administrators of a domain (moderators) may choose to interfere in editorial conflicts, and may suspend or ban individuals for behaviour they consider inappropriate.
+COSS resolves natural conflicts between teams and vendors by allowing anyone to define a new specification.
+There is no editorial control process except that practised by the editor of a new specification.
+The administrators of a domain (moderators) may choose to interfere in editorial conflicts,
+and may suspend or ban individuals for behaviour they consider inappropriate.
+
+
+## Specification Structure
+
+### Meta Information
+
+Specifications MUST contain following metadata.
```

Comment:
```suggestion
Specifications MUST contain the following metadata.
```

Reply:
Do we want to increase this to a MUST? Seems kinda nice to not _have to_ include things like tags

Reply:
The tags are optional: `0 or several tags`. I could make this more explicit in the text.
Imo, header information (except for tags) should be a `MUST`. 
Informational RFCs do not have to follow a specific structure, but marking them as informational + listing the responsible editor should be a `MUST`, imo.
Wdyt?